app/api/torznab/api.py

```
@@ -259,14 +281,187 @@ def torznab_api(
+                    )
+                    try:
+                        magnet = tn.build_magnet(
+                            title=release_title,
+                            slug=slug,
+                            season=season_i,
+                            episode=ep_i,
+                            language=lang,
+                            provider=prov,
+                            site=site_found,
+                        )
                     except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)